<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monatsansicht</class>
 <widget class="QDialog" name="Monatsansicht">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 248, 248);</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>541</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(8, 8, 8);
border-top-color: rgb(255, 255, 255);
background-color: rgb(234, 247, 255);
selection-color: rgb(255, 255, 255);
selection-background-color: rgb(0, 0, 127);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="termin_erstellen">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>805</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
  
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}</string>
   </property>
   <property name="text">
    <string>Termin hinzufügen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="faecher">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>805</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
  
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}</string>
   </property>
   <property name="text">
    <string>Studifächer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profil">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
  
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}</string>
   </property>
   <property name="text">
    <string>Profil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lernplan">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>805</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
  
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}</string>
   </property>
   <property name="text">
    <string>Lernplan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Das steht heute an:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Datelabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ansicht">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>200</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
  border: 2px solid gray;
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Monatsansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tagesansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wochenansicht</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="monat_vor">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>300</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="monat_zurueck">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>300</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>315</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView
{
  border: 2px solid rgb(0, 0, 127);
  border-radius: 10px;
  padding: 0 8px;
background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>491</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTableView
{
  border: 2px solid rgb(0, 0, 127);
  border-radius: 10px;
  padding: 0 8px;
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Abmelden">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>990</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
  
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
}</string>
   </property>
   <property name="text">
    <string>Abmelden</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>1921</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(234, 247, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(234, 247, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l5">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l6">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l7">
   <property name="geometry">
    <rect>
     <x>1680</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>920</y>
     <width>248</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="terminlabel">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>275</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1870</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 22, 25);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackBackround">
   <property name="geometry">
    <rect>
     <x>1799</x>
     <y>89</y>
     <width>66</width>
     <height>66</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(171, 171, 171);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="profilePicture">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>90</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>calendarWidget</zorder>
  <zorder>termin_erstellen</zorder>
  <zorder>faecher</zorder>
  <zorder>profil</zorder>
  <zorder>lernplan</zorder>
  <zorder>label</zorder>
  <zorder>DateLabel</zorder>
  <zorder>ansicht</zorder>
  <zorder>tableView</zorder>
  <zorder>tableView_2</zorder>
  <zorder>label_2</zorder>
  <zorder>Abmelden</zorder>
  <zorder>line</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>l1</zorder>
  <zorder>monat_zurueck</zorder>
  <zorder>monat_vor</zorder>
  <zorder>l2</zorder>
  <zorder>l3</zorder>
  <zorder>l4</zorder>
  <zorder>l5</zorder>
  <zorder>l6</zorder>
  <zorder>l7</zorder>
  <zorder>label_6</zorder>
  <zorder>logo</zorder>
  <zorder>terminlabel</zorder>
  <zorder>exit</zorder>
  <zorder>blackBackround</zorder>
  <zorder>profilePicture</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
